Python/optimizer_bytecodes.c

```diff
@@ -337,41 +339,47 @@ dummy_func(void) {
     }
 
     op(_TO_BOOL, (value -- res)) {
-        if (!optimize_to_bool(this_instr, ctx, value, &res)) {
+        int opt = optimize_to_bool(this_instr, ctx, value, &res);
```

Comment:
What's the rationale behind this change? (just curious) Is it for some optimization reason or just to fit on 80 chars for PEP 7?